diff --git a/Cargo.lock b/Cargo.lock
index df31171..9305bfe 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -2191,7 +2191,7 @@ checksum = "c19937216e9d3aa9956d9bb8dfc0b0c8beb6058fc4f7a4dc4d850edf86a237d6"
 [[package]]
 name = "libdrm_amdgpu_sys"
 version = "0.8.4"
-source = "git+https://github.com/Umio-Yasuno/libdrm-amdgpu-sys-rs?rev=69196107b2a8ab8dabfd67265f004f16f146a308#69196107b2a8ab8dabfd67265f004f16f146a308"
+source = "git+https://github.com/Dr4kk3N/libdrm-amdgpu-sys-rs?rev=69196107b2a8ab8dabfd67265f004f16f146a308#69196107b2a8ab8dabfd67265f004f16f146a308"
 dependencies = [
  "libc",
  "libloading",
diff --git a/crates/libamdgpu_top/Cargo.toml b/crates/libamdgpu_top/Cargo.toml
index e1a5631..6888dc3 100644
--- a/crates/libamdgpu_top/Cargo.toml
+++ b/crates/libamdgpu_top/Cargo.toml
@@ -16,5 +16,5 @@ libdrm_dynamic_loading = ["libdrm_amdgpu_sys/dynamic_loading"]
 
 [dependencies]
 anyhow = { version = "1.0" }
-libdrm_amdgpu_sys = { version = "0.8.4", git = "https://github.com/Umio-Yasuno/libdrm-amdgpu-sys-rs", default-features = false, rev = "69196107b2a8ab8dabfd67265f004f16f146a308" }
+libdrm_amdgpu_sys = { version = "0.8.4", git = "https://github.com/Dr4kk3N/libdrm-amdgpu-sys-rs", default-features = false, rev = "69196107b2a8ab8dabfd67265f004f16f146a308" }
 nix = { version = "0.29", default-features = false, features = [ "ioctl" ] }
